### PR TITLE
[V1] VLM - enable processor cache by default

### PR DESCRIPTION
**Pull request by alexm-redhat**
We have found that V1 VLM prefix caching performs best when it is combined with VLM processor cache. This PR enables VLM processor cache by default, so we get best performance in all cases.

### CODE REVIEW

**Comment by comaniac** [comment]
You should also deal with `--mm-cache-preprocessor` I think. It's not just `store true` so will become useless after this PR.

**Comment by alexm-neuralmagic** [comment]
@comaniac good catch! I changed --mm-cache-preprocessor to be --disable-mm-cache-preprocessor

**Review by comaniac — APPROVED** [review]
LGTM

**Comment by mgoin** [comment]
Why not keep `--mm-cache-preprocessor` and just change default value to True? It seems unnecessary to rename a boolean value

**Comment by alexm-neuralmagic** [comment]
Just wanted to be consistent with that we already have, like --disable-log-requests or --disable-async-output-proc

File: examples/offline_inference_vision_language.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def run_aria(question: str, modality: str):
               tokenizer_mode="slow",
               trust_remote_code=True,
               dtype="bfloat16",
-              mm_cache_preprocessor=args.mm_cache_preprocessor)
+              disable_mm_cache_preprocessor=args.disable_mm_cache_preprocessor)
```
Comment (ywang96) [inline examples/offline_inference_vision_language.py:31]: Just a very small nit: do you know it makes sense to rename this arg to `disable_mm_preprocessor_cache`? I think the naming is more intuitive in that way

**Review by ywang96 — APPROVED** [review]
Thanks for making this PR!

File: examples/offline_inference_vision_language.py (hunk 1 above)
Comment (alexm-neuralmagic) [reply examples/offline_inference_vision_language.py:31]: yeah, will rename to disable_mm_preprocessor_cache

**Pull request merged**